.appveyor.yml

```diff
@@ -0,0 +1,58 @@
+version: '{build}'
+
+skip_commits:
+  files:
+    - README.md
+    - .travis.yml
+
+os: Visual Studio 2017
+
+environment:
+  matrix:
+#    - Configuration: Debug
+#      SDK: orangebox
+#    - Configuration: Debug BMS
+#      SDK: bms
+#    - Configuration: Debug 2013
+#      SDK: sdk2013
+#    - Configuration: Debug OE
+#      SDK: episode1
+#    - Configuration: Debug P2
+#      SDK: portal2
+    - Configuration: Release
+      SDK: orangebox
+    - Configuration: Release BMS
+      SDK: bms
+    - Configuration: Release 2013
+      SDK: sdk2013
+    - Configuration: Release OE
+      SDK: episode1
+    - Configuration: Release P2
+      SDK: portal2
+
+install: |
+  git submodule update --init --recursive
+  git clone --depth 1 -b %SDK% https://github.com/alliedmodders/hl2sdk.git
+  mkdir hl2sdk\utils\SourcePauseTool
+  move * hl2sdk\utils\SourcePauseTool
+  move SDK hl2sdk\utils\SourcePauseTool
+  move SPTLib hl2sdk\utils\SourcePauseTool
+  move Signals hl2sdk\\utils\SourcePauseTool
+  move spt hl2sdk\utils\SourcePauseTool
```

Comment:
Instead of listing every folder, what if you did something like
```
cd ..
git clone --depth 1 -b %SDK% https://github.com/alliedmodders/hl2sdk.git
mkdir hl2sdk\utils
move SourcePauseTool hl2sdk\utils
cd hl2sdk\utils\SourcePauseTool
```

Reply:
@YaLTeR Done.